projects.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: contact
+---
+
```

Comment:
This page looks like this:

![Screenshot 2022-03-30 at 09-17-07 Scala Center at EPFL](https://user-images.githubusercontent.com/332812/160775519-d6c4ca48-60e4-46db-a5c7-0cd09011fd25.png)